packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.read-replica.js.snapshot/cdk-rds-read-replica.template.json

```diff
@@ -206,28 +206,28 @@
      "Ref": "PostgresSourceSubnetGroupBEEB1740"
     },
     "Engine": "postgres",
-    "EngineVersion": "15.2",
-    "MasterUsername": {
+    "EngineVersion": "16.2",
+    "MasterUserPassword": {
```

Comment:
Curious why did `MasterUsername` get changed to `MasterUserPassword`. Doesn't seem to see any related changes in the integ test file.

Reply:
Oh nvm, on a more detail look, it seems like this is also just re-ordering which confused me because it moves `MasterUserPassword` to top and `MasterUsername` to bottom.